tools/run-dev.py

```diff
@@ -404,7 +404,7 @@ def shutdown_handler(*args, **kwargs):
     for s in (signal.SIGINT, signal.SIGTERM):
         signal.signal(s, shutdown_handler)
     ioloop.start()
-except:
+except Exception:
```

Comment:
I think it's OK to change this, since it just controls the printing; the `finally` happens regardless.

https://docs.python.org/3/tutorial/errors.html#defining-clean-up-actions